### DISCUSSION TITLE
Curation report January 23-27

### DISCUSSION THREAD

**Discussion by danidi** [wikipathways/GPML-Repo#110] [Curation reports]
I saw that the user Conroy lipids added several ECs to the interactions as comments. Do we have a better place than comments to store this information?

https://www.wikipathways.org/index.php/Pathway:WP5317 looks like a new test pathway, but has the description "do not modify or delecte" as it is used by the wikipathways academy. Is this really the case?

One question regarding descriptions: I vaguely remember that we shouldn't use Wikipedia text directly due to Wikipathways being CC0 while Wikipedia has a stricter license. Is this still the case? If yes, I would revert @AlexanderPico's change on https://www.wikipathways.org/index.php/Pathway:WP1020.

https://www.wikipathways.org/index.php/Pathway:WP5308 gives a "sorry the tags cannot be fetched!" error.

Search works.

@elissonnog why was https://www.wikipathways.org/index.php/Pathway:WP5291 tagged for deletion?

Next one is @fehrhart.

**Comment by khanspers** [comment]
Thanks Daniela! 

For WP5317, it is a test pathway. The comment in the Description is left from the starter pathway ([WP3908](https://www.wikipathways.org/index.php/Pathway:WP3908). I added the "Test pathway" tag to WP5317.

Regarding descriptions re-used verbatim from Wikipedia, my understanding is that this is ok if attribution is given, for example by linking to the original article. See https://en.wikipedia.org/wiki/Wikipedia:Reusing_Wikipedia_content#Text_content.

**Reply by egonw** [reply]
> Regarding descriptions re-used verbatim from Wikipedia, my understanding is that this is ok if attribution is given

That used to be the case before we switched CCZero. We are now back to "fair use": short quotes is okay, but longer text snippets not.

**Comment by AlexanderPico** [comment]
And I copied the description from the parent Human pathway ([WP357](https://www.wikipathways.org/index.php/Pathway:WP357)) from which this Cow version ([WP1020](https://www.wikipathways.org/index.php/Pathway:WP1020)) was created via homology mapping. So, updates to the description should go into both of these, as well as the other approved versions:
 - [WP504](https://www.wikipathways.org/index.php/Pathway:WP504)
 - [WP38](https://www.wikipathways.org/index.php/Pathway:WP38)
 - [WP336](https://www.wikipathways.org/index.php/Pathway:WP336)
 - [WP568](https://www.wikipathways.org/index.php/Pathway:WP568)

**Comment by AlexanderPico** [comment]
Also worth noting... a blank Description will throw an error in our new system. That's how WP1020 came to my attention.

**Comment by danidi** [comment]
Thank you, @AlexanderPico for the information on the other pathways! @egonw, are three sentences still regarded as short quote, or should we write something else?

**Reply by egonw** [reply]
> are three sentences still regarded as short quote, or should we write something else?

I am not an expert in "fair use". I strongly suggest to rewrite them.

**Reply by AlexanderPico** [reply]
[ChatGPT](https://chat.openai.com/chat) can do this sort of task:

My prompt: "Rewrite these sentences to avoid plagiarism: ..."

Their response: "The production of fatty acids from acetyl-CoA and NADPH through the use of enzymes known as fatty acid synthases is referred to as fatty acid synthesis. This biological process occurs within the cytoplasm of cells. The majority of acetyl-CoA that is transformed into fatty acids originates from carbohydrates through the glycolytic pathway."

I'll paste this in to all 6 GPMLs now...

**Reply by AlexanderPico** [reply]
Done!

**Reply by Chris-Evelo** [reply]
Fun :-)

**Reply by danidi** [reply]
Nice, thank you!

**Comment by elissonnog** [comment]
Hello @danidi 
I did it because the pathway was created in October (2022) and stayed without update after that.

**Reply by danidi** [reply]
Thank you for the explanation! However, the content seems to be original, but needs substantial work (e.g. better title and description, literature references, database identifiers. From these aspects, it is similar to https://www.wikipathways.org/index.php/Pathway:WP5290 from the same author, which is actually approved. I removed the deletion tag and added a needs work tag with this description. Does someone know the author kozo2 and could contact him to ask about the plans for these pathways?

**Reply by khanspers** [reply]
Thanks @danidi . I will contact Kozo to see what the plan for the pathway is.

**Reply by danidi** [reply]
Thank you! I already sent him a message on Slack, Egon gave me his contact yesterday. But I don't know if he still uses Slack.
